dashboard/test/models/course_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class CourseTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
I've found commenting out tests to be one of the most effective ways of getting them to pass.